src/components/form/contact.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { ChangeEventHandler, FormEventHandler, useState } from 'react';
+import styled from 'styled-components';
+import { Text } from '../typography/typography';
+
+const InputContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
By default, flex-direction is always row, so this is a bit redundant.

Reply:
Go catch, will remove it :)